manager/integration/tests/test_csi_snapshotter.py

```diff
@@ -1188,3 +1188,304 @@ def finalizer():
                               'default')
 
     request.addfinalizer(finalizer)
+
+
+def test_csi_volumesnapshot_backing_image_basic():
+    """
+    Test Create/Delete BackingImage using VolumeSnapshot with a given Volume
+
+    Setup
+    - Create a VolumeSnapshotClass with type `bi`
+        ```
+        kind: VolumeSnapshotClass
+        apiVersion: snapshot.storage.k8s.io/v1
+        metadata:
+            name: longhorn-snapshot-vsc
+        driver: driver.longhorn.io
+        deletionPolicy: Delete
+        parameters:
+            type: bi
+        ```
+
+    Given
+    - A Volume `test-vol` attached to a workload
+
+    When
+    - Creating the VolumeSnapshot
+        ```
+        apiVersion: snapshot.storage.k8s.io/v1beta1
+        kind: VolumeSnapshot
+        metadata:
+            name: test-snapshot-backing
+        spec:
+            volumeSnapshotClassName: longhorn-snapshot-vsc
+            source:
+                persistentVolumeClaimName: test-vol
+        ```
+
+    Then
+    - A BackingImage is created with the following properties
+        ```
+        apiVersion: longhorn.io/v1beta2
+        kind: BackingImage
+        metadata:
+            name: `snapshot-${VolumeSnapshot.uuid}`
+            namespace: longhorn-system
+        spec:
+            sourceType: export-from-volume
+            sourceParameters:
+                volume-name: test-vol
+                export-type: raw
+        ```
+
```

Comment:
We can add a check to verify the data.

Reply:
I will merge 
`test_csi_volumesnapshot_backing_image_basic`
`test_csi_volumesnapshot_backing_image_restore` into one

So the flow will be
create volume -> create snapshot -> create volume from snapshot -> verify data
all in the same test